apache/superset
# Update to fix the broken blueprints link

The current link is http://flask.pocoo.org/docs/0.12/blueprints/ which redirects to a non-existent page. The new link is https://flask.palletsprojects.com/en/1.1.x/tutorial/views/.

### CATEGORY

Choose one

- [ x ] Documentation

### SUMMARY

Update to the broken link for redirection to flask blueprints.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TEST PLAN
1. Go to superset's installation page and scroll to the blueprints section https://superset.incubator.apache.org/installation.html#blueprints
2. Click on the link on the first sentence; "Blueprints are Flask’s reusable apps"
3. The link should redirect to Flask's blueprint documentation at https://flask.palletsprojects.com/en/1.1.x/tutorial/views/

### ADDITIONAL INFORMATION



### REVIEWERS
